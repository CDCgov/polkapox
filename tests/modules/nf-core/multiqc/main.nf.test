nextflow_process {

    name "Test Process MULTIQC"
    script "modules/nf-core/multiqc/main.nf"
    process "MULTIQC"

    test("Multiqc Test") {

        when {
            params {

            }
            process {
                """
                input[0] = file("${projectDir}/tests/modules/nf-core/multiqc")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
