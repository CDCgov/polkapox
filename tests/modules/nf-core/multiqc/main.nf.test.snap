{
    "Multiqc Test": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "2": [
                    
                ],
                "3": [
                    
                ],
                "data": [
                    
                ],
                "plots": [
                    
                ],
                "report": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-07-08T14:40:12.71410351"
    }
}